### PR TITLE
Barne2sj code review

### PR DESCRIPTION
**Pull request by barne2sj**
Analysis of program: The program seems to run correctly. It appears to be a system to add items to an inventory. The UI isn't great yet, but there's still time to work on that. It looks like the team is planning on implementing Bootstrap but has yet to do so. Also, for some reason there are 3 location fields.

Was the program available on GitHub on time?: Yes

Is the program easy to understand?: For the most part. There aren't very many comments yet, but I could get a sense of what the code was doing. As a user, though, it is a little confusing with the multiple location fields. I was unsure what the difference was between them.

Does the program compile?: Yes

Links to 3 commits I made on my group's repo during Sprint 1: 
https://github.com/AustinGarrison/Ent-App-Development/commit/98f7e314005b88a347934401b72809f097b1b5c6
https://github.com/AustinGarrison/Ent-App-Development/commit/025c073680e7ccaf4bd10d2e675f80aef43169fb
https://github.com/AustinGarrison/Ent-App-Development/commit/a271c9aceadf5dd73efeafe676ad7b8a3692b589

### CODE REVIEW

File: src/main/java/edu/uc/inventorymanager/ui/MainController.java [hunk 1]
```diff
@@ -40,7 +40,6 @@ public String index(Model model) {
 
     @RequestMapping("/saveItem")
     public String saveItem(Item item) {
-        //itemService.save(item);
```
Comment (barne2sj) [inline src/main/java/edu/uc/inventorymanager/ui/MainController.java:43]: Removed commented out code to improve quality of code. This helps code be more readable and easier to maintain, reducing technical debt. Source: https://martinfowler.com/bliki/TechnicalDebt.html

File: src/main/java/edu/uc/inventorymanager/dto/Item.java [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 @RequiredArgsConstructor
 public @Data class Item {
-    @NonNull private int id;
-    private String name;
+    @NonNull private int itemId;
+    private String itemName;
```
Comment (barne2sj) [inline src/main/java/edu/uc/inventorymanager/dto/Item.java:10]: Made variable names more specific so it is more clear what they are representing. This is a principle of clean code. Source: https://www.baeldung.com/java-clean-code

File: src/main/resources/templates/index.html [hunk 3]
```diff
@@ -8,11 +8,11 @@
     <title>Inventory Manager</title>
 
 
-    <!--Dan's stuff-->
+    <!--Item form-->
```
Comment (barne2sj) [inline src/main/resources/templates/index.html:11]: Made comments clearer so they are more useful to people viewing the code. This is a principle of clean code. Source: https://www.baeldung.com/java-clean-code

File: src/main/java/edu/uc/inventorymanager/dto/Item.java (hunk 2 above)
Comment (discospiff) [reply src/main/java/edu/uc/inventorymanager/dto/Item.java:10]: I'd leave this as it were.
The context "Item" is a given, based on the class name, so adding "item" to each variable is redundant.

File: src/main/java/edu/uc/inventorymanager/dto/User.java [hunk 4]
```diff
@@ -3,6 +3,6 @@
 import lombok.Data;
 
 public @Data class User {
-    private int id;
-    private String name;
+    private int userId;
+    private String username;
```
Comment (discospiff) [inline src/main/java/edu/uc/inventorymanager/dto/User.java:7]: Ditto comment above.

File: src/main/java/edu/uc/inventorymanager/ui/MainController.java [hunk 5]
```diff
@@ -24,7 +24,7 @@
     /**
      * Handle the / endpoint
      *
-     * @return
+     * @return index
```
Comment (discospiff) [inline src/main/java/edu/uc/inventorymanager/ui/MainController.java:27]: As long as you're editing @return, give a description of what it is returning.  What can a caller expect when invoking this function?

File: src/main/java/edu/uc/inventorymanager/ui/MainController.java [hunk 6]
```diff
@@ -50,18 +49,7 @@ public String saveItem(Item item) {
...
-    @ResponseBody
-    public Item createItem(@ResponseBody Item item){
-        Item newItem = null;
-        try {
-            //newItem = itemService.save(item);
-        } catch (Exception e){
-            //
-        }
-    }
-     */
```
Comment (discospiff) [inline src/main/java/edu/uc/inventorymanager/ui/MainController.java:64]: Good.

Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.

File: src/main/java/edu/uc/inventorymanager/ui/MainController.java [hunk 7]
```diff
@@ -73,7 +61,7 @@ public Item createItem(@ResponseBody Item item){
      * 400: item not found
      *
      * @param id
-     * @return
+     * @return HttpStatus
```
Comment (discospiff) [inline src/main/java/edu/uc/inventorymanager/ui/MainController.java:64]: Add description

File: src/main/resources/templates/index.html [hunk 8]
```diff
@@ -22,12 +22,12 @@
...
         <input type="text" name="status_id" th:field="*{statusId}"/><br><br>
         <label>Item ID</label>
-        <input type="text" name="id" th:field="*{id}"/><br><br>
+        <input type="text" name="id" th:field="*{itemId}"/><br><br>
         <button name="submit" type="submit" value="0">Submit</button>
     </form>
 
 
-    <!--End of Dan's stuff-->
+    <!--End of Item Form-->
```
Comment (discospiff) [inline src/main/resources/templates/index.html:30]: Probably don't need this at all.  Remove it.

**Comment by discospiff** [comment]
What I'd harvest from this review: 
Remove commented out code.

**Pull request closed without merging**